content/influxdb/v2.0/get-started.md

```diff
@@ -334,6 +334,32 @@ _To run InfluxDB in [detached mode](https://docs.docker.com/engine/reference/run
        quay.io/influxdb/influxdb:v2.0.4
    ```
 
+### Configure InfluxDB with Docker
+
+To mount an InfluxDB configuration file and use it from within Docker:
+
+1. follow the steps above in ["Persist data outside the InfluxDB container"](#persist-data-outside-the-influxdb-container).
+
+2. Use the command below to generate the default configuration file on the host file system:
+
+    ```console
+    $ docker run \
+        --rm quay.io/influxdb/influxdb:v2.0.4 \
+        influxd print-config > config.yml
+    ```
+
+3. Modify the default configuration, which will now be available under `$PWD`.
+
+4. Then start the InfluxDB container:
+
+   ```console
+   $ docker run -p 8086:8086 \
+         -v $PWD/config.yml:/etc/influxdb2/config.yml:ro \
```

Comment:
I've copied this line [from here](https://github.com/docker-library/docs/pull/1871/files#diff-447ed53aad4d938ca4937b1dea577b7f3d226609d99d1be13fe63e7929ce9276R101). What does this `:ro` mean? @danxmoran

Reply:
It mounts the file as read-only. It's not really required, but a nice sanity-check